src/main/preprocessing/code_tracker_handler.py

```diff
@@ -15,7 +15,7 @@
 
 
 def fill_column(data: pd.DataFrame, column: consts.CODE_TRACKER_COLUMN,
-                default_value: consts.DEFAULT_VALUES) -> Union[Any, consts.DEFAULT_VALUES]:
+                default_value: consts.DEFAULT_VALUE) -> Union[Any, consts.DEFAULT_VALUE]:
     values = data[column].unique()
     index = np.argwhere((values == default_value) | pd.isnull(values))
```

Comment:
I think we should use `default_value.value`, because of df contains a value, not Enum class. Also, it likely requires tests. It was my part and you can note it **TODO** or you can do it yourself